src/tests/fileIOSuite.c

```diff
@@ -0,0 +1,112 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+#ifdef _WIN32
+#include <fcntl.h>
+#include <io.h>
+#endif
+
+#include <string.h>
+
+#include "CuTest.h"
+#include "fileIO.h"
+
+#define TEST_STRING "Lorem ipsum dolor sit amet, consectetur adipiscing elit.\nMorbi dignissim nunc vel lectus egestas, nec rutrum nibh gravida.\n\t(){}[];:<>\\\"'"
+
+/* When writing to a file and reading the file to a string, it should be the same */

```

Comment:
I would either remove these comments, or rewrite them a bit to be clearer and distribute them to the methods testing each file.

Reply:
The comments have been rewritten these comments and placed them by their respective functions